package.json

```diff
@@ -399,10 +399,16 @@
 		"request": "^2.88.2",
 		"request-promise": "^4.2.6",
 		"shelljs": "^0.8.5",
-		"strip-ansi": "^6.0.1",
+		"strip-ansi": "^7.0.1",
 		"tar": "^6.1.11",
 		"tmp": "^0.2.1",
 		"valid-filename": "^3.1.0",
 		"vscode-kubernetes-tools-api": "^1.3.0"
+	},
+	"__metadata": {
+		"id": "5181e15d-d18d-444e-ba3b-e0cf48cab6f8",
+		"publisherDisplayName": "Red Hat",
+		"publisherId": "eed56242-9699-4317-8bc7-e9f4b9bdd3ff",
+		"isPreReleaseVersion": false
```

Comment:
why is it added?

Reply:
I'm not sure. I'll remove it and try if it breaks anything. It might be some leftover of some node script.